src/Raven.Studio/typescript/viewmodels/resources/manageDatabaseGroup.ts

```diff
@@ -39,6 +39,7 @@ class manageDatabaseGroup extends viewModelBase {
     private sortable: any;
 
     private graph = new databaseGroupGraph();
+    backupsOnly = false;
```

Comment:
?

Reply:
the fix for the ongoing tasks graph legend...

Reply:
ok